package.json

```diff
@@ -279,6 +279,7 @@
     "@types/react": "18.3.12",
     "browserify-sign": "4.2.2",
     "devtools-protocol": "0.0.1413303",
+    "node-abi": "3.74.0",
```

Comment:
See this note as to why this was necessary: https://github.com/cypress-io/cypress/pull/31260/files#diff-43831bef862964fca0c4c1b68381da0539c3892c9b356e7db2a19cf25cfc4295R90